frontend/src/components/Input/index.module.css

```diff
@@ -0,0 +1,54 @@
+.input {
+  padding: 20px 24px;
+  font-family: "Work Sans", sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 20px;
+  color: #000;
+  border: 1px solid #9f18d9;
+  border-radius: 10px;
+  box-shadow: 0 4px 5px #9f18d9 0.5;
+}
+
+.input:active {
+  color: #000;
+  outline-style: none;
+}
+
+.input:focus {
+  outline-style: none;
+}
+
+.input:disabled {
+  color: #404040;
+  background: #bfbfbf;
+  border: 1px solid #404040;
+  border-radius: 10px;
+  box-shadow: 0 4px 5px #808080;
+}
+
+.inputError {
+  border-color: red;
+}
+
+.container {
+  display: inline-grid;
+  font-size: 12px;
+  line-height: 16px;
+  color: #F73F52;
+}
+
+::placeholder {
+  color: #BFBFBF;
+}
```

Comment:
What do you think this does?

Reply:
It change the color of the placeholder text

Reply:
So I believe this style will leak into other components if we don't prefix it with `.input`